### ISSUE TITLE
Graphics develops a haloing effect around text and icons

### ISSUE THREAD

**Issue by kprawl** [files-community/Files#14578]
### Description

I just paid for and install from Microsoft Store. The app looks great when I first start it up but then line rings start to show around texts and icons and keep multiplying and expanding until it is nearly impossible to read and see what is on the screen. I have an HP Envy 17M Laptop with an Intel i7 7500U 7th Gen processor and a Nvidia GeForce 950MX video card. It has 16GB of Ram. I feel like it is perhaps a customization setting somewhere that is causing this but not sure.
![20240129_131911](https://github.com/files-community/Files/assets/158089865/ca940b42-1ff4-40e6-8f12-e606042caf56)

### Steps To Reproduce

Start the app and let it set. After about 15 seconds the rings start appearing and multiplying.

### Requirements

I just need it to stay clear so that it can be read and used.

### Files Version

3.1.1.0

### Windows Version

Edition	Windows 10 Home Version	22H2 Installed on	‎7/‎24/‎2020 OS build	19045.3930 Experience	Windows Feature Experience Pack 1000.19053.1000.0

### Log File

[debug.log](https://github.com/files-community/Files/files/14088586/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, Can you send a screenshot via `Windows key + Shift + S` as your picture is very low quality.

**Comment by kprawl** [comment]
No. Only this app

**Comment by kprawl** [comment]
Like I said, when I first open it up it looks normal, then it starts making those rings around everything.

**Comment by kprawl** [comment]
It's only on the menu items. The File area is clear.

**Comment by kprawl** [comment]
![image](https://github.com/files-community/Files/assets/158089865/b2100d0a-f48c-41f8-a36e-a69fa5116e31)
Eventually it looks like this. Can hardly read.

**Comment by kprawl** [comment]
Yes. It does the same thing in the Preview Files app also.

**Comment by kprawl** [comment]
![image](https://github.com/files-community/Files/assets/158089865/8537b41e-a6c3-4bcc-b7c1-0418a6d1f62f)
In the Preview Files app it doesn't start out clear. It start with these halos then grows out, but at a slower rate.

**Comment by yaira2** [comment]
Wow! This is definitely an issue with WinAppSdk but I'll see if there are any solutions.

**Comment by yaira2** [comment]
@kprawl can you try changing backdrop material to Acrylic?
![image](https://github.com/files-community/Files/assets/39923744/58df9b6e-6d05-4e80-ad88-7808d328713e)

**Comment by kprawl** [comment]
I tries that earlier. Still did the same thing. Another thing I noticed was when the app was starting up your folder logo also started transforming with a similar pattern.

⁣Get TypeApp for Android ​

**Comment by yaira2** [comment]
What about changing it to "none"? To do this you'll need to manually edit the settings file.
1. Open Files > Settings > Advanced > Edit settings file. This will open the settings file in your default text editor.
2. Before editing the settings file, you’ll need to close Files by right clicking the icon in the system tray and clicking on "Quit".
3. The settings file has a JSON format. You can use any text editor to open and modify it.
4. Change the value of `AppThemeBackdropMaterial` to `0`.
5. Save the settings file and restart the app.

**Comment by kprawl** [comment]
Changed according to your instructions and it still did not fix the problem.

**Comment by yaira2** [comment]
Thanks for checking

**Comment by kprawl** [comment]
I have fixed the problem. You have a conflict with the Nvidia Graphics Card evidently. I went into the Nvidia Control Panel and changed a setting in the Manage 3D settings from High-Performance NVIDIA processor to Integrated graphics and no longer have the haloing effect. After Files worked I went back into those settings and changed it back and now it still worked. But when I did the QUIT button and opened Files it reverted back to the haloing. Hope this information will help you fix this.
